phpunit.xml.dist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/7.5/phpunit.xsd"
```

Comment:
Well, the current schema used should be compatible with PHPUnit 4.x - 9.2. The schema changed in PHPUnit 9.3, but using the new schema is (obviously) not an option if we want the tests to keep running on older PHPUnit versions.

Once PHPUnit 10.x comes out, the CI may need the "on-the-fly" update of the XML file using `--migrate-configuration`.

So with all this in mind, I just went with a version which is sort of in the middle of all supported PHPUnit versions.

Re: using the online scheme vs the one in vendor - the one in `vendor` will change depending on the PHP version used to run `composer install` as the most appropriate PHPUnit version will be installed, so that doesn't feel like a reliable solution.

Reply:
Agreed.